stakingModel/test/UBIScheme.test.js

```diff
@@ -166,23 +174,24 @@ contract(
     await identity.addWhitelisted(claimer4);
     let claimer3BalanceBefore = await goodDollar.balanceOf(claimer3);
     let claimer4BalanceBefore = await goodDollar.balanceOf(claimer4);
-    await ubi.distribute([claimer3, claimer4]);
+    await ubi.distribute(0,10);
     let claimer3BalanceAfter = await goodDollar.balanceOf(claimer3);
     let claimer4BalanceAfter = await goodDollar.balanceOf(claimer4);
     expect(claimer3BalanceAfter.toNumber()).to.be.equal(claimer3BalanceBefore.toNumber());
     expect(claimer4BalanceAfter.toNumber()).to.be.equal(claimer4BalanceBefore.toNumber());
   });
 
-  it("should not be able to execute claim after the user received tokens since distribute had been executed", async () => {
-    let claimer4BalanceBefore = await goodDollar.balanceOf(claimer4);
-    await ubi.claim({ from: claimer4 })
+  it("should be able to execute successfully claim after the user had received tokens as a result of distribute which had been executed", async () => {
```

Comment:
dont understand this test

Reply:
changed the description c7586741b9c670251e051b652b23e1d9ab534969